docs/docs/references/preview-projects.mdx

```diff
@@ -0,0 +1,173 @@
+import CreatePreview from './assets/create-preview.jpg';
+import DeletePreview from './assets/delete-preview.jpg';
+
+# Preview Projects
+
+**Preview projects** are temporary Lightdash projects where you can safely experiment with your metrics, dimensions and charts without affecting your production project. 
+
+Preview projects are commonly used to:
+- test changes in models, metrics and dimensions.
+- make changes to important content in production without any downtime by [promoting the content](/guides/how-to-promote-content) you've changed in your preview project when it's ready.
+
+## Creating a preview project
+
+:::info
+
+Only users who have developer permissions or above in a project can create preview projects.
+
+:::
+
+### Lighdash CLI Tool (Local)
+
+You can use the [Lightdash CLI tool](/guides/cli/how-to-install-the-lightdash-cli) to generate preview projects in Lightdash from your command line. 
+
+```shell
+# This will create a preview and will wait until you press a key to delete the preview project
+lightdash preview
+```
+
+or
+
+```shell
+# This will create a preview and exit, you will have to run lightdash stop-preview to delete it
+lightdash start-preview
+```
+
+You can see more information about the [`lightdash preview` CLI command here](/references/lightdash-cli#lightdash-preview).
+
+### Lightdash App
+
+You can create a preview project in the Lightdash web app by clicking on the projects list, then clicking `+ Create Preview`. 
```

Comment:
@TuringLovesDeathMetal what is the source project in this case? Just the project I'm currently in when I push that button?

Reply:
You can pick the source project in the modal!

Reply:
updated the docs image ✅